src/integration/kotlin/pl/edu/agh/gem/integration/ability/ServiceTestClient.kt

```diff
@@ -30,4 +30,10 @@ class ServiceTestClient(applicationContext: WebApplicationContext) {
             .headers { it.withValidatedUser(user) }
             .exchange()
     }
+
+    fun generateGroupAttachment(userId: String, groupId: String): ResponseSpec {
+        return webClient.post()
+            .uri(URI("/internal/groups/$groupId/users/$userId/generate"))
+            .exchange()
```

Comment:
shouldn't we add header: .withAppAcceptType() ?

Reply:
sure we can add that but that header with only check on client side